algorithms/hhl/hhl.ipynb

```diff
@@ -0,0 +1,1367 @@
+{
```

Comment:
<p>"For simplicity, the demo below treats a usecase where... demo":</p><p>You have already stated that b is normalized and A is of size 2^n\times 2^n. I suggest the following modification of this sentence:</p><p>"For simplicity, the demo below treats a usecase where the matrix A has eigenvalues in the interval (0,1). Generalizations to other usecases, including the case where |b|\neq 1 and A is not an an Hermitian matrix of size 2^n \times 2^n, are discussed at the end of this demo".</p><p> </p><p>Concerning speedup, you can also write that it assumes an efficient loading for the state b.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Added assumption about the initial state loading and modified info about the demo.